Header header 
float32[] centroid_points
float32[] voronoi_polygons
float32[] main_polygon
